pkg/contract/contract.go

```diff
@@ -93,18 +111,22 @@ func (sc Contract) Verify() error {
 	}
 
 	amsterdamLocation, _ := time.LoadLocation(AmsterdamTimeZone)
-	now := NowFunc()
-
-	if now.In(amsterdamLocation).Before(*validFrom) {
+	if checkTime.In(amsterdamLocation).Before(*validFrom) {
```

Comment:
Some tests indicating this is not broken would be nice

Reply:
https://github.com/nuts-foundation/nuts-auth/blob/6849f0de67d0adf8dbe5693803964ca23fcc39dc/pkg/contract/contract_test.go#L105

Reply:
Seems covered by the parsing